hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/container/common/helpers/StorageContainerException.java

```diff
@@ -100,5 +100,4 @@ public ContainerProtos.Result getResult() {
     return result;
   }
 
-
```

Comment:
Normally we try to avoid whitespace-only changes in unrelated files.  In order to save CI time, I'm OK with it, unless someone requests other changes, too.